engine-rest/engine-rest-openapi/src/main/templates/paths/history/job-log/count/get.ftl

```diff
@@ -0,0 +1,51 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/job-log/get-job-log-query-count/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricJobLogsCount"
+      tag = "Historic Job Log"
+      summary = "Get Job Log Count"
+      desc = "Queries for the number of historic job logs that fulfill the given parameters.
+              Takes the same parameters as the
+              [Get Job Logs](${docsUrl}/reference/rest/history/job-log/get-job-log-query/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = true >
+    <#assign requestMethod="GET"/>
```

Comment:
I cannot see that we use the request method in the `history-job-log-params.ftl`. Should we remove it here? It also does no harm as far as I can tell but I guess it's also not necessary?